types/header.d.ts

```diff
@@ -2,3 +2,155 @@
  * The header type declaration of `undici`.
  */
 export type IncomingHttpHeaders = Record<string, string | string[] | undefined>;
+
+type HeaderNames =
```

Comment:
I think they should be lowercased?

Reply:
they are in the map, so it auto completes both the cased and lower cased values